specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2018-05-05/AlertsManagement.json

```diff
@@ -5,6 +5,7 @@
     "title": "Azure Alerts Management Service Resource Provider",
     "description": "Azure Alerts Management Service provides a single pane of glass of alerts across Azure Monitor."
   },
+  "host": "management.azure.com",
```

Comment:
ah host is not strictly required in the swagger def, but I tink we can treat is as it should be in hte semantic validator, open a bug ?

Reply:
Yes was thinking the same thing.  Open in https://github.com/azure/oav?

Reply:
yup

Reply:
Opened https://github.com/Azure/oav/issues/413